Update benchmark.md (#124)
statechannels/SAFE-protocol: doc/benchmark.md

  - For nxtp, each swap is independent (there is no batching), so 1 token type is used.
- The benchmarks avoid “cold” stores (modifying a zero storage value to non-zero). So the contracts and swap addresses are primed to hold tokens.
- We assume a gas cost of 150 gwei for mainnet as this was approximately the average gas cost on **January 17th**

> - We assume a cost of 0.001 gwei for Optimism's L2gas cost, and also include the dominant L1gas cost for Optimism rollup data to be posted on L1. 

Reviewer: ```suggestion
- We assume a cost of 0.001 gwei for Optimism's L2 gas cost.
- The dominant cost for an Optimism transaction is the cost of rollup data posted to L1. This cost depends on L1 gas cost. 
```

|               | NXTP                                                                                    | SAFE                                                                                 |
| ------------- | --------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------ |

> | L1 cost(gwei) | 21798000 gas \* 150 gwei = 3,269,700,000 gwei                                              | 1,150,678 gas \* 150 gwei = 172,601,700 gwei                                           |

Reviewer: ```suggestion
| L1 cost(gwei) | 21798000 L1 gas \* 150 gwei = 3,269,700,000 gwei                                              | 1,150,678 gas \* 150 gwei = 172,601,700 gwei                                           |
```

> | L1 cost(gwei) | 21798000 L1 gas \* 150 gwei = 3,269,700,000 gwei                                              | 1,150,678 gas \* 150 gwei = 172,601,700 gwei                                           |

Reviewer: ```suggestion
| L1 cost(gwei) | 21798000 L1 gas \* 150 gwei = 3,269,700,000 gwei                                              | 1,150,678 L1 gas \* 150 gwei = 172,601,700 gwei                                           |
```

> | L2 cost(gwei) | (2,756,000 L1gas \* 150 gwei + 18,545,900 L2gas \* 0.001 gwei) \* 1.24 ~= 512,638,996 gwei | (502,376 L1gas \* 150 gwei + 16,830,846 L2gas \* 0.001 gwei) \* 1.24 ~= 93,462,806 gwei |

Reviewer: ```suggestion
| L2 cost(gwei) | (2,756,000 L1 gas \* 150 gwei + 18,545,900 L2 gas \* 0.001 gwei) \* 1.24 ~= 512,638,996 gwei | (502,376 L1 gas \* 150 gwei + 16,830,846 L2 gas \* 0.001 gwei) \* 1.24 ~= 93,462,806 gwei |
```
